catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -128,4 +171,21 @@ protected SolrCacheMBean getCacheProxy() {
 
         return metacards;
     }
+
+    private java.util.List<Result> getResultList(int amount) {
+        java.util.List<Result> results = new ArrayList<>();
+
+        for (int i = 0; i < amount; i++) {
+
+            String id = UUID.randomUUID()
+                    .toString();
+            MetacardImpl metacard = new MetacardImpl();
```

Comment:
Can a mock work here instead of MetacardImpl and ResultImpl instances? If changed to use mocks, make sure to add `when` clauses before adding to list.